apps/etterlatte-beregning/src/main/kotlin/beregning/BarnepensjonAnvendtTrygdetidPerioder.kt

```diff
@@ -0,0 +1,112 @@
+package no.nav.etterlatte.beregning
+
+import no.nav.etterlatte.beregning.grunnlag.BeregningsGrunnlag
+import no.nav.etterlatte.beregning.grunnlag.GrunnlagMedPeriode
+import no.nav.etterlatte.beregning.grunnlag.kombinerOverlappendePerioder
+import no.nav.etterlatte.beregning.regler.AnvendtTrygdetid
+import no.nav.etterlatte.beregning.regler.barnepensjon.trygdetidsfaktor.TrygdetidGrunnlag
+import no.nav.etterlatte.beregning.regler.barnepensjon.trygdetidsfaktor.anvendtTrygdetidRegel
+import no.nav.etterlatte.beregning.regler.toSamlet
+import no.nav.etterlatte.libs.common.beregning.SamletTrygdetidMedBeregningsMetode
+import no.nav.etterlatte.libs.common.feilhaandtering.InternfeilException
+import no.nav.etterlatte.libs.common.feilhaandtering.UgyldigForespoerselException
+import no.nav.etterlatte.libs.common.logging.sikkerlogger
+import no.nav.etterlatte.libs.common.trygdetid.TrygdetidDto
+import no.nav.etterlatte.libs.regler.FaktumNode
+import no.nav.etterlatte.libs.regler.KonstantGrunnlag
+import no.nav.etterlatte.libs.regler.PeriodisertResultat
+import no.nav.etterlatte.libs.regler.RegelPeriode
+import no.nav.etterlatte.libs.regler.RegelkjoeringResultat
+import no.nav.etterlatte.libs.regler.eksekver
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+
+object BarnepensjonAnvendtTrygdetidPerioder {
```

Comment:
Må vi ikke også lagre ned tilsvarende for OMS?

Reply:
Jo, det kan det fort hende vi bør.